job-dsls/jobs/srcclr_scan_job.groovy

```diff
@@ -0,0 +1,81 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def repoFile = new URL(Constants.REPO_FILE_URL).text
+def repoList = repoFile.readLines()
+
+for (repo in repoList) {
+
+    def jobName = "srcclr-scan-${repo}"
+
+    job(jobName) {
+
+        description("Job responsible for SourceClear verification of ${repo}")
+
+        parameters {
+            choiceParam('SCAN_TYPE', ['scm', 'binary'])
+            stringParam('SRCCLR_INVOKER_REPO_URL','')
+            stringParam('URL','')
+            stringParam('VERSION', '')
+            stringParam('PACKAGE','')
+            stringParam('NAME', '')
+            stringParam('MVNPARAMS', '')
+            choiceParam('PROCESSOR_TYPE', ['cve', 'cvss'])
+            booleanParam('RECURSIVE', false)
+            booleanParam('DEBUGGING', false)
+            booleanParam('TRACING', false)
+            stringParam('SCMVERSION', '')
+            stringParam('THRESHOLD', '1','Threshold from 1 to 10 for cvss processor')
+        }
+
+
+        environmentVariables{
+            groovy('''
+          def map = [:]
+          map.put("RECURSE", Boolean.valueOf("${RECURSIVE}") ? "--recursive" : "")
+          map.put("DEBUG", Boolean.valueOf("${DEBUGGING}") ? "-d" : "");
+          map.put("TRACE", Boolean.valueOf("${TRACING}") ? "--trace" : "")
+          map.put("MVNPARAMETER", "${MVNPARAMS}" !="" ? "--maven-param=${MVNPARAMS}":"")
+          map.put("SCMVERSIONPARAM", "${SCAN_TYPE}" == "scm" ? " --ref=${SCMVERSION}":"")
+          return map
+        ''')
+        }
+
+
+        label("kie-rhel7")
```

Comment:
again why not `& !master`

Reply:
Because it was starting slowly. I deliberately made this commit: https://github.com/kiegroup/kie-jenkins-scripts/pull/726/commits/8880ee65df2b0cb983534f686e6f2a3f34c2b367
Now when I saw it was working, I will put it back.

Reply:
And here it is back again: https://github.com/kiegroup/kie-jenkins-scripts/pull/726/commits/70173f767e8898efe59b977753fda7c7d7ce0abd